lib/api/playlist.js

```diff
@@ -0,0 +1,123 @@
+const ApiResource = require('./api_resource')
+const PageableApiResource = require('./pageable_api_resource')
+const qs = require('qs')
+
+/**
+ * @class Playlist
+ */
+class Playlist extends ApiResource {
+  /**
+   * Retrieves flag if playlist is available for a specific channel.
+   *
+   * @param {Number}    channelId
+   *
+   * @returns {*}
+   */
+  isEnabled (channelId) {
+    return this.context.authRequest('get', `/channels/${channelId}/settings/playlists.json`)
+      .then((res) => {
+        return Promise.resolve(res.is_enabled)
```

Comment:
All variables should be in `camelCase`. `is_enabled` should be renamed to `isEnabled`.